Tasks/AzureAppServiceManageV0/azureappservicemanage.ts

```diff
@@ -13,6 +13,37 @@ import { KuduServiceUtils } from './operations/KuduServiceUtils';
 import { AzureResourceFilterUtils } from './operations/AzureResourceFilterUtils';
 import { enableContinuousMonitoring } from './operations/ContinuousMonitoringUtils';
 
+const webAppKindMap = new Map([
+    [ 'app', 'webApp' ],
+    [ 'app,linux', 'webAppLinux' ],
+    [ 'app,container', 'webAppContainer']
+]);
+
+async function advancedSlotSwap(updateDeploymentStatus: boolean, targetSlot: string, appServiceSourceSlot: AzureAppService, appServiceTargetSlot: AzureAppService, appServiceSourceSlotUtils: AzureAppServiceUtils, appServiceTargetSlotUtils: AzureAppServiceUtils, preserveVnet: boolean, caseValue: string) {
+
+    if(appServiceSourceSlot.getSlot().toLowerCase() == appServiceTargetSlot.getSlot().toLowerCase()) {
+        updateDeploymentStatus = false;
+        throw new Error(tl.loc('SourceAndTargetSlotCannotBeSame'));
+    }
+
+    console.log(tl.loc('WarmingUpSlots'));
+    try {
+        await Promise.all([appServiceSourceSlotUtils.pingApplication(), appServiceTargetSlotUtils.pingApplication()]);
+    }
+    catch(error) {
+        tl.debug('Failed to warm-up slots. Error: ' + error);
+    }
+    
+    if(caseValue == "Swap Slots" || caseValue == "Complete Swap") {
+        await appServiceSourceSlot.swap(targetSlot, preserveVnet);
```

Comment:
These you should call in respective cases. i.e. swap for action swap slots, complate swap